Extensions/satsukimous.js

```diff
@@ -67,6 +67,9 @@ XKit.extensions.satsukimous = new Object({
 		}
 		$( "img" ).filter(function( index ) {
 			return $( this ).attr( "src" ).indexOf( "anonymous_avatar" ) !== -1;
+		}).each(function ( index ) {
+			$( this ).data("origsrc", $( this ).attr( "src" ));
```

Comment:
should be `function(index) {` 

(or `function( index ) {` if you want to match the rest of the code, but I'd prefer removing the extra whitespace and adding a linter check for it....)